### DISCUSSION TITLE
Scene e automazioni

### DISCUSSION THREAD

**Discussion by mriitan** [indomus/forum#1378] [HUB: Home Assistant]
Ciao a tutti, mi scuso per la banailtà delle domande, mi sto approcciando e sto testando con una versione su imac HA per la mia casa.
Sono riuscito a integrare quasi tutti gli elementi, ora il problema è che vorrei creare delle sequenze di azioni su quelli che però non sono dispositivi ma entità, ho visto su diverse guide che le scene possono essere formate da dispositivi ed entità, ma la nella mia demo sono presenti solo i dispositivi. come mai?
Banalmente vorrei chiudere tutte le tapparelle della zona notte, quindi far eseguire tramite sonoff queste operazione alle 5 tapparelle in parallelo, cose che con google home ed una routine faccio regolarmente, qualche suggerimento?
grazie mille

**Comment by Scialla** [comment]
Le entità le vedi su lovelace da configurazione, dispositivi e servizi, entità, oppure premendo la lettera "e" sulla tastiera.
Per fare quello che dici non serve una scena ma un banale interruttore "dummy" e una automazione. Se attivi l'interruttore finto fa partire la chiusura di tutte le tapparelle e viceversa.
Crea un input_boolean (finto interruttore) e crea una automazione che usa questo come trigger

**Reply by mriitan** [reply]
si ok, ma per le automazioni, il sistema vede solo i dispositivi e non le entità, come procedo? grazie

**Reply by snakuzzo** [reply]
automazioni, script e scene vengono visti come routine...non sono dispositivi (giustamente)
Le trovi quando andrai a creare le routine su alexa o google assistant.

**Reply by mriitan** [reply]
scusami, forse mi esprimo male, quando vado a creare una automazione, tramite il configuratore, alla scelta degli oggetti da utilizzare, il sonoff (nello specifico ad esempio alza tapparella balcone, il canale del sonoff che alza la tapparella se on), questo canale è visto come entità e non come dispositivo e quindi tra gli oggetti non lo trovo.
grazie

**Reply by snakuzzo** [reply]
pensavo ti riferissi agli assistenti vocali.
Ad ogni modo...i dispositivi sono composti "anche" da entità.
Non so cosa tu stia facendo, ma se da GUI vai sulla sezione di creazione delle scene troverai la possibilità sia di aggiungere device che entità.
![acq](https://user-images.githubusercontent.com/7205371/150752036-cec68405-2e2d-4449-b8d6-bfea883e9b4f.png)

**Reply by mriitan** [reply]
ok, perfetto, nelle scene vedo le entità, che posso aggiungere e mettere in parallelo per far compiere una determinata azione.
Per far scattare una scena al premere di un pulsante devo aggiungere un trigger? grazie sempre

**Reply by snakuzzo** [reply]
sì, però perdonami...leggi la documentazione e qualche guida...
es.

https://indomus.it/formazione/le-automazioni-di-home-assistant-cosa-sono-e-come-si-usano/

**Reply by mriitan** [reply]
hai perfettamente ragione, solo che mi sto perdendo un pò, devo fare il punto e chiarire alcune cose
grazie comunque, gentilissimo

**Comment by mriitan** [comment]
allora vediamo se qualcuno può confermarmi il tutto.
Ho creato un commutatore via GUI con Aiutanti.
Ho creato quindi una automazione, che funziona in Modalità SINGOLO, e si attiva con Tipo di Attivazione: Stato, sulla Entità Commutatore creato in precedenza che cambia stato da Spento ad ACCESO per 10 secondi.
Le azioni che dovranno essere eseguite sono un SERVIZIO: Interruttore Turn ON delle entità switch che mi interessano.
Ho quindi inserito in plancia un Pulsante che modifichi lo stato del mio aiutante ma l'automazione non parte, l'automazione dalla sezione di creazione e prova funziona correttamente, ma con il tasto in Plancia non si attiva.

**Reply by snakuzzo** [reply]
Fai vedere il codice YAML dell'automazione.
Se non sai dove si trova è nel file `automations.yaml`

**Reply by mriitan** [reply]
![image](https://user-images.githubusercontent.com/98173792/150785523-cf8d24e2-3a6a-4565-ad04-ad4e2b0ba8eb.png)
eccolo qui

**Reply by snakuzzo** [reply]
Dovresti copiare e incollare il codice usando il markdown, per cortesia.
Immagini di testo NON si possono vedere

**Reply by mriitan** [reply]
```
- id: '1643022774242'
  alias: Auto Abbassa Zona Giorno
  description: ''
  trigger:
  - platform: state
    entity_id: input_boolean.abbassa_zona_giorno
    from: SPENTO
    to: ACCESO
  condition: []
  action:
  - service: switch.turn_on
    target:
      entity_id: switch.sonoff_10005b5c85_1
  mode: single

```

**Reply by snakuzzo** [reply]
l'input boolean non avrà mai gli stati `ACCESO` e `SPENTO`... quelli li vedi così su lovelace in base alla lingua impotsata...ma lo stato reale è un altro :) 
gli stati sono `'on'` e `'off'`. Per vedere queste cose ti tocca guardare un po' di documentazione e (consiglio da amico) "giocare" con gli strumenti sviluppatore ;)

**Reply by mriitan** [reply]
cavolo ho provato Off e On, invece serve tutto minuscolo.... ho visto ora negli strumenti sviluppatore.... grazie per la pazienza
